zipline/testing/core.py

```diff
@@ -465,31 +327,25 @@ def make_trade_data_for_asset_info(dates,
     volumes = (volume_sid_deltas + volume_date_deltas[:, None]) + volume_start
 
     for j, sid in enumerate(sids):
-        start_date = asset_info[sid]['start_date']
-        end_date = asset_info[sid]['end_date']
+        start_date = asset_info.loc[sid, 'start_date']
```

Comment:
I also sometimes write this specific pattern as `start_date, end_date = df.loc[sid, ['start_date', 'end_date']]`.